ttnn/cpp/ttnn/operations/moreh/moreh_linear_backward/moreh_linear_backward.cpp

```diff
@@ -0,0 +1,175 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "moreh_linear_backward.hpp"
+
+#include "ttnn/cpp/ttnn/deprecated/tt_dnn/op_library/moreh_helper_functions.hpp"
+#include "ttnn/cpp/ttnn/operations/moreh/moreh_matmul/moreh_matmul.hpp"
+#include "ttnn/cpp/ttnn/operations/moreh/moreh_sum/moreh_sum.hpp"
+
+namespace ttnn::operations::moreh::moreh_linear_backward {
+
+std::tuple<bool, bool, bool> MorehLinearBackward::get_required_outputs(const std::vector<bool>& are_required_outputs) {
+    if (are_required_outputs.size() != 3) {
+        TT_ASSERT(are_required_outputs.size() == 3, "are_required_outputs size must be 3");
```

Comment:
```suggestion
        TT_FATAL(are_required_outputs.size() == 3, "are_required_outputs size must be 3");
```

Reply:
resolved in [9b40fca](https://github.com/tenstorrent/tt-metal/pull/12812/commits/9b40fca6fe2a092b7588eaeac2d24d998ecb517d)